acm_rules_vm.tf

```diff
@@ -0,0 +1,56 @@
+# ------------------------------------------------------------------------------
+# Create an ACM certificate for rules.vm.cyber.dhs.gov.
+# ------------------------------------------------------------------------------
+
+resource "aws_acm_certificate" "rules" {
+  provider = aws.acmresourcechange
+
+  domain_name       = "rules.vm.cyber.dhs.gov"
+  validation_method = "DNS"
+
+  lifecycle {
+    create_before_destroy = true
+  }
+
+  options {
+    certificate_transparency_logging_preference = "ENABLED"
+  }
+}
+
+# ------------------------------------------------------------------------------
+# Create public DNS records containing validation information for the
+# ACM certificate created above.
+# ------------------------------------------------------------------------------
+
+resource "aws_route53_record" "rules_certificate_validation" {
+  provider = aws.route53resourcechange
+
+  for_each = {
+    for dvo in aws_acm_certificate.rules.domain_validation_options : dvo.domain_name => {
+      name   = dvo.resource_record_name
+      record = dvo.resource_record_value
+      type   = dvo.resource_record_type
+    }
+  }
+
+  allow_overwrite = true
+
+  depends_on = [aws_acm_certificate.rules]
```

Comment:
Is this strictly necessary? Since the resource is used in the `for_each` it should be an implicit dependency right?

Reply:
Good catch - I cribbed that code from somewhere else and didn't think to remove that unnecessary `depends_on`.  That has been rectified in https://github.com/cisagov/cool-dns-cyber.dhs.gov/pull/69/commits/ec0bf559a2a986be3a523a6bd79060c8a7377795.